Description: fix "open with" actions
Last-Update: 2019-07-26

--- rootactions-servicemenu-2.7.3+p8+git20160617.0159.orig/usr/bin/rootactions-servicemenu.pl
+++ rootactions-servicemenu-2.7.3+p8+git20160617.0159/usr/bin/rootactions-servicemenu.pl
@@ -992,11 +992,11 @@ sub do_open_with {
 	   
 	   # Fixed bug with qt5 applications not opening
            # because they don't provide the kde-options like "--caption" any more
-           $OPTTEST = system "$CPROGRAM --help-kde" ;
-           if ( $OPTTEST eq 0 ) {
-              exec "$DBUSRUN $CPROGRAM --caption \"$APPNAME $ROOTREMINDER\" \'$TARGET\'" ; }
-           else {
-              exec "KDE_SESSION_VERSION=5 KDE_FULL_SESSION=true $DBUSRUN $CPROGRAM \'$TARGET\'" ; }
+#           $OPTTEST = system "$CPROGRAM --help-kde" ;
+#           if ( $OPTTEST eq 0 ) {
+#              exec "$DBUSRUN $CPROGRAM --caption \"$APPNAME $ROOTREMINDER\" \'$TARGET\'" ; }
+#           else {
+              exec "KDE_SESSION_VERSION=5 KDE_FULL_SESSION=true $DBUSRUN $CPROGRAM \'$TARGET\'" ;
            exit $?;
         }
 }
